tests/integration/models_images_test.py

```diff
@@ -87,8 +87,14 @@ def test_pull_with_sha(self):
     def test_pull_multiple(self):
         client = docker.from_env(version=TEST_API_VERSION)
         images = client.images.pull('hello-world')
-        assert len(images) == 1
-        assert 'hello-world:latest' in images[0].attrs['RepoTags']
+        assert len(images) >= 1
+        found = False
+        for img in images:
+            if 'hello-world:latest' == img.attrs['RepoTags']:
```

Comment:
Hi @thaJeztah 

Could you try use unicode for image name, like `u'hello-world:latest'`

Reply:
Sure, I can try that (would that be a bit odd though, given that image references only support a-z0-9 ? 🤔 or does python return it as unicode? (I'm not really a Python expert 😅)

Reply:
Python 2 doesn't use Unicode as default.

Also, I forgot that `img.attrs['RepoTags']` is a list, so you can change the comparator `==` to `in` and the test will pass.

Reply:
Doh! You're right, completely forgot about that; yes, that should be the problem

Reply:
Yeah, Unicode wasn't that bad guy this time. hahaha